test/e2e/run_selinux_test.go

```diff
@@ -165,4 +165,16 @@ var _ = Describe("Podman run", func() {
 		Expect(session.ExitCode()).To(Equal(126))
 	})
 
+	It("podman exec selinux check", func() {
+		setup := podmanTest.RunTopContainer("test1")
+		setup.WaitWithDefaultTimeout()
+		Expect(setup.ExitCode()).To(Equal(0))
+
+		session := podmanTest.Podman([]string{"exec", "test1", "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		session1 := podmanTest.Podman([]string{"exec", "test1", "cat", "/proc/self/attr/current"})
+		session1.WaitWithDefaultTimeout()
+		Expect(session.OutputToString()).To(Equal(session1.OutputToString()))
```

Comment:
I'm not sure I understand this test; I get the same results with 1.6.2-2.fc30 (i.e. two consecutive `exec`s have the same context).  Just mentioning it as an FYI.

Reply:
Oops typo.  First one is supposed to be /proc/1/attr/current, Will open a PR to fix.